src/ert/gui/model/snapshot.py

```diff
@@ -347,28 +362,30 @@ def data(self, index: QModelIndex, role=Qt.DisplayRole):
         if role == Qt.TextAlignmentRole:
             return Qt.AlignCenter
 
-        node = index.internalPointer()
+        node: Union[RootNode, IterNode, RealNode, ForwardModelStepNode] = (
+            index.internalPointer()
+        )
 
         if role == NodeRole:
             return node
 
         if role == IsEnsembleRole:
-            return node.type == NodeType.ITER
+            return isinstance(node, IterNode)
         if role == IsRealizationRole:
-            return node.type == NodeType.REAL
+            return isinstance(node, RealNode)
         if role == IsJobRole:
-            return node.type == NodeType.JOB
+            return isinstance(node, ForwardModelStepNode)
 
-        if node.type == NodeType.JOB:
+        if isinstance(node, ForwardModelStepNode):
             return self._job_data(index, node, role)
-        if node.type == NodeType.REAL:
+        if isinstance(node, RealNode):
             return self._real_data(index, node, role)
 
         if role == Qt.DisplayRole:
             if index.column() == 0:
-                return f"{node.type}:{node.id}"
+                return f"{type(node)}:{node.id_}"
```

Comment:
Is it tested somewhere that `type(node)` gives the same string as before? (not sure how/if this is displayed to the user)

Reply:
I cannot figure out where it is displayed, but when printing it; it looked a bit different.


**Before**:
```
NodeType.REAL:0
NodeType.ITER:0
NodeType.ITER:0
NodeType.JOB:0
NodeType.JOB:0
NodeType.ITER:0
NodeType.REAL:0
NodeType.ITER:0
NodeType.ITER:0
```
**After**:
```
<class 'ert.gui.model.node.IterNode'>:0
<class 'ert.gui.model.node.IterNode'>:0
<class 'ert.gui.model.node.ForwardModelStepNode'>:0
<class 'ert.gui.model.node.ForwardModelStepNode'>:0
<class 'ert.gui.model.node.ForwardModelStepNode'>:0
<class 'ert.gui.model.node.ForwardModelStepNode'>:0
<class 'ert.gui.model.node.RealNode'>:0
<class 'ert.gui.model.node.RealNode'>:0
<class 'ert.gui.model.node.IterNode'>:0
<class 'ert.gui.model.node.IterNode'>:0
<class 'ert.gui.model.node.ForwardModelStepNode'>:0
<class 'ert.gui.model.node.ForwardModelStepNode'>:0
<class 'ert.gui.model.node.IterNode'>:0
<class 'ert.gui.model.node.RealNode'>:0
<class 'ert.gui.model.node.IterNode'>:0
<class 'ert.gui.model.node.IterNode'>:0
```

Is this something worth cleaning up? I do not see the purpose of displaying this to the user anyways.

Reply:
I can atleast add the type(...)._\_name_\_

Reply:
^ this looked better
```
IterNode:0
ForwardModelStepNode:0
RealNode:0
RealNode:0
ForwardModelStepNode:0
RealNode:0
RealNode:0
IterNode:0
IterNode:0
ForwardModelStepNode:0
IterNode:0
```

Reply:
Turns out this is printed to the terminal after this PR was merged..